src/lib/convenience/ContextAttributeResponseVector.cpp

```diff
@@ -69,18 +74,19 @@ std::string ContextAttributeResponseVector::render(ConnectionInfo* ciP, RequestT
 */
 std::string ContextAttributeResponseVector::check
 (
-  ConnectionInfo*  ciP,
-  RequestType      request,
-  std::string      indent,
-  std::string      predetectedError,
-  int              counter
+  const std::string&  apiVersion,
+  bool                asJsonObject,
+  RequestType         request,
+  std::string         indent,
```

Comment:
Now that we change, let's try to change this as well:

    const std::string&   (x2)